games-emulation/dosboxx/additional-files/dosbox.rdef

```diff
@@ -0,0 +1,116 @@
+
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+       major  = 0,
+       middle = 7,
+       minor  = 4,
```

Comment:
Where did this version number come from?

Reply:
Ops, you are correct, this file was copied from DOSBox and I forgot to update it